src/main/java/com/artipie/settings/repo/CrudRepoSettings.java

```diff
@@ -28,27 +28,48 @@ public interface CrudRepoSettings {
 
     /**
      * Check if the repository exists.
-     * @param name Repository name. The name can be composite: in the case of org layout it will
-     *  consist of two parts - username and repo name, for example john/maven-s3.
+     * @param rname Repository name, for example maven-s3.
      * @return True if repository exists
      */
-    boolean exists(String name);
+    boolean exists(String rname);
```

Comment:
I do not think this interface should know anything about combined (with username) or not repository names. This logic should be processed on the upper level, here already finalized repository name should be passed